src/inference/src/system_conf.cpp

```diff
@@ -225,6 +225,7 @@ std::vector<int> get_available_numa_nodes() {
 #    endif
 
 std::vector<std::vector<int>> get_proc_type_table() {
+    std::lock_guard<std::mutex> lock{cpu._cpu_mutex};
     return cpu._proc_type_table;
```

Comment:
It seems also need add lock to protect all read/write ov::CPU::cpu data, such as: `is_cpu_map_available()` and `get_num_numa_nodes()`, to avoid potential risk in the future.

Reply:
Yes, you are right. I need to add lock before the variable that may be modified.